src/Esfa.Vacancy.Register.Application/Queries/SearchApprenticeshipVacancies/SearchApprenticeshipVacanciesRequestValidator.cs

```diff
@@ -14,26 +14,31 @@ public SearchApprenticeshipVacanciesRequestValidator()
         {
             RuleFor(request => request.StandardCodes)
                 .NotEmpty()
-                .When(request => !request.FrameworkCodes.Any())
-                .WithMessage(MinimumFieldsErrorMessage);
+                .When(request => request.FrameworkCodes == null || !request.FrameworkCodes.Any())
```

Comment:
Null check is not required, all the collections are initialized to avoid null checks.

Reply:
So I can't remember how I came across needing this null check now, could have just been unit tests, but I don't think it's harmful as the intention of the query handler is serve more than just the orchestrator, so maybe we will have weird requests coming in.. I'm not bothered either way other than its in there now.